cli/args/mod.rs

```diff
@@ -1129,6 +1130,25 @@ impl CliOptions {
     }
   }
 
+  pub fn otel_config(&self) -> Option<OtelConfig> {
+    if self
+      .flags
+      .unstable_config
+      .features
+      .contains(&String::from("otel"))
+    {
+      Some(OtelConfig {
+        default_service_name: Cow::Borrowed("deno"),
+        default_service_version: Cow::Borrowed(
+          crate::version::DENO_VERSION_INFO.deno,
+        ),
+        ..Default::default()
```

Comment:
This shouldn't be the service name. We should set these the keys `process.runtime.name` and `process.runtime.version`: https://opentelemetry.io/docs/specs/semconv/resource/process/#javascript-runtimes